forms/tests/conftest.py

```diff
@@ -0,0 +1,264 @@
+        field_type=basic_field_types["textbox"],
+    )
+    field_factory(
+        label="Personal identity code",
+        section=contact_person_company_applicant_section,
+        field_type=basic_field_types["textbox"],
+    )
+
+    # Person applicant
+    person_applicant_section = section_factory(
+        form=form, parent=applicant_section, title="Person information", visible=False
+    )
+    field_factory(
+        label="First name",
+        section=contact_person_company_applicant_section,
+        field_type=basic_field_types["textbox"],
+    )
+    field_factory(
+        label="Last name",
+        section=contact_person_company_applicant_section,
+        field_type=basic_field_types["Textbox"],
+    )
+    field_factory(
+        label="Personal identity code",
+        section=contact_person_company_applicant_section,
+        field_type=basic_field_types["textbox"],
+    )
+
+    # Subsection for person's contact person information
+    person_contact_person_section = section_factory(
+        form=form, parent=person_applicant_section, title="Contact person"
+    )
+    field_factory(
+        label="Different than applicant",
+        section=person_contact_person_section,
+        field_type=basic_field_types["checkbox"],
+        action="ToggleEnableInSection=" + person_contact_person_section.identifier,
+    )
+    field_factory(
+        label="First name",
+        section=person_contact_person_section,
+        field_type=basic_field_types["textbox"],
+        enabled=False,
+    )
+    field_factory(
+        label="Last name",
+        section=person_contact_person_section,
+        field_type=basic_field_types["Textbox"],
+        enabled=False,
+    )
+    field_factory(
+        label="Personal identity code",
+        section=person_contact_person_section,
+        field_type=basic_field_types["textbox"],
+        enabled=False,
+    )
+
+    additional_info_applicant_section = section_factory(
+        form=form, parent=applicant_section, title="Additional information",
+    )
+    field_factory(
+        label="Additional information",
+        section=additional_info_applicant_section,
+        field_type=basic_field_types["textarea"],
+    )
+
+    # Applicant switcher: Company / Person
+    applicant_type_switcher_field = field_factory(
+        section=applicant_section, field_type=basic_field_types["radiobuttoninline"],
+    )
+    choice_factory(
+        field=applicant_type_switcher_field,
+        text="Company",
+        value="1",
+        action="ShowSection="
+        + company_applicant_section.identifier
+        + ", HideSection="
+        + person_applicant_section.identifier,
+    )
+    choice_factory(
+        field=applicant_type_switcher_field,
+        text="Person",
+        value="2",
+        action="ShowSection="
+        + person_applicant_section.identifier
+        + ", HideSection="
+        + company_applicant_section.identifier,
+    )
+
+    # Root searched target section
```

Comment:
Not all too important as this is ""only"" a variable name in tests, but `searched` is IMHO not the best translation for `haettava`/`haettu`. Sadly I don't have great suggestion to replace it either... I'd say it should be more something like "bid"  or "request for proposals" or "call for applications" or "competition" (as in tonttikilpailu).

Reply:
This is changed to "Application target"